crytic_compile/platform/etherscan.py

```diff
@@ -97,6 +97,21 @@
     "sepoliaera.zksync": ("300", "sepolia-era.zksync.network"),
     "xai": ("660279", "xaiscan.io"),
     "sepolia.xai": ("37714555429", "sepolia.xaiscan.io"),
+    "xdc": ("50", "https://xdcscan.com"),
```

Comment:
`https://` should not be on these lists (mostly for consistency with the existing entries, I'm not sure we need the URL for any v2 operation). You can generate an updated list with the function down below, `generate_supported_network_v2_list`

```python
from crytic_compile.platform.etherscan import generate_supported_network_v2_list
generate_supported_network_v2_list()
```